release_notes/markdown.py

```diff
@@ -0,0 +1,152 @@
+import dataclasses
+import logging
+import typing
+from collections import defaultdict
+
+import release_notes.model as rnm
+
+logger = logging.getLogger(__name__)
+
+
+@dataclasses.dataclass
+class Title:
+    display: str
+    identifiers: list[str]
+
+    def __hash__(self):
+        return hash(self.display)
+
+    def __eq__(self, other):
+        return (
+            other is not None
+            and isinstance(other, Title)
+            and (other == self or hash(other) == hash(self))
+        )
+
+
+def _simple_title(display: str) -> Title:
+    return Title(display, [display.lower()])
+
+
+categories: list[Title] = [
```

Comment:
preferably use `tuple` + does typehint actually add much value here? I am not strong opposed to keeping typehint; however, considering this is a "readonly" "define", the type should be obvious from the value..